README.md

```diff
@@ -27,3 +27,11 @@ and self-esteem by navigating the life of their avatar!
 ## Documentation of PowerUp (Android)
 Here's the link to the official documentation:
 [Visit Documentation!](http://chetnagsocpowerupandroid.blogspot.in/2015/05/database-design.html)
+
+##Coding Guidelines
+1. Don't use magic numbers or hard-coded strings. Put them in dimens.xml or strings.xml
+2. Class names should be in CamelCase. Name activities with names including Activity so it's easier to know what they are.
```

Comment:
You can use `1.` for all elements in the list and that will generate the right order so changes are easier